cg/store/api/status.py

```diff
@@ -103,6 +103,24 @@ def cases_to_store(self, pipeline: str, limit: int = None) -> list:
         )
         return list(families_query)[:limit]
 
+    def orders_to_microsalt_analyze(self):
```

Comment:
```suggestion
    def get_orders_to_analyze_with_microsalt(self):
```

Reply:
add return type

Reply:
since Patrik's PRs I can use the existing  `cases_to_store` instead, so I removed this